### PR TITLE
Fix html errors

### PR DESCRIPTION
**Pull request by binfalse**
there were quite a few HTML issues...

### CODE REVIEW

File: src/main/java/widoco/LODEParser.java [hunk 1]
```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright 2012-2013 Ontology Engineering Group, Universidad Polit�cnica de Madrid, Spain
+ * Copyright 2012-2013 Ontology Engineering Group, Universidad Polit�cnica de Madrid, Spain
```
Comment (dgarijo) [inline src/main/java/widoco/LODEParser.java]: I see a possible encoding error here. It doesn't show the accent :S

File: src/main/java/widoco/Constants.java [hunk 2]
```diff
@@ -552,7 +562,7 @@ public static String getIndexDocument(String resourcesFolderName,Configuration c
                     "			j++;\n" +
                     "			t+= '<li>'+(i-1)+'.'+j+'. '+'<a href=#'+ jQuery(this).attr('id')+'>'+ jQuery(this).text()+'</a></li>';\n" +
                     "		}\n" +
-                    "		t = t.replace(' "+lang.getProperty(LANG_BACK3).replace("&iacute;", "�")+"','');\n" +//back to ToC
+                    "		t = t.replace(' "+lang.getProperty(LANG_BACK3).replace("&iacute;", "�")+"','');\n" +//back to ToC
```
Comment (dgarijo) [inline src/main/java/widoco/Constants.java]: I cannot see the accent properly here.

**Comment by dgarijo** [comment]
Thanks a lot for taking the time to do this. There are a couple of accents that I cannot see correctly, I wonder if the encoding used is compatible (or similar). Could you please have a look (i have left comments).
Also, if you have more pull requests could you please do them on the develop branch? I use master just for releases.

**Comment by binfalse** [comment]
Sorry, character encoding across systems is an evergreen ;-)
Therefore I don't understand why you're replacing the proper HTML `&iacute;` for an error-prone `í`? Wouldn't it be better to have the `&iacute;`?

Tried to fix the accents with fb5e454 - does is now work for you?

**Comment by dgarijo** [comment]
Looks good to me! Thanks!
I probably did the replacement as a quick fix because it was showing up incorrectly in the final html. I cannot remember though. I have opened an issue (#256) to remind me having a look

**Pull request merged**